.travis.yml

```diff
@@ -50,6 +50,17 @@ script:
   - xdebug-enable
   - vendor/bin/phpunit --coverage-clover=build/logs/clover.xml
   - xdebug-disable
+  - if [[ "$WITH_PHAR" == "true" ]]; then composer box; fi
 
 after_success:
   - php composer/bin/test-reporter
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_API_KEY
```

Comment:
This needs to be provided either in the Travis settings or as an encrypted environment variable, I believe.